packages/web/src/lib/wallectconnect-mode.ts

```diff
@@ -39,13 +39,22 @@ export class WallectConnectSession {
     this.wcAccounts = this.connector.accounts;
   }
 
+  /**
+   * Kills a session if already exists, create new session
+   */
+  async forceCreate (): Promise<void> {
+    if (this.connector.connected) {
+      try { await this.close(); } catch (e) {};
+    }
+    return await this.create();
+  }
+
   /**
    * Create new session
    */
   async create (): Promise<void> {
     if (this.connector.connected) {
       console.warn(`A session is already active`);
-      return;
```

Comment:
Why you delete this line?

Reply:
good question, probably we should  return a current session if possible.

Reply:
right, this shouldn't be removed. thanks